centinel/client.py

```diff
@@ -37,7 +37,7 @@ def get_input_file(self, experiment_name):
         return os.path.join(self.config['dirs']['data_dir'], input_file)
 
     def load_input_file(self, name):
-        input_file = self.get_input_file(name)
+        input_file = self.get_input_file("%s.txt" %(name))
```

Comment:
By the way, I think my previous response didn't bring this out, but your current solution will only accept input files that end in .txt We need to either support the previous way (no extension) or be extension agnostic. Changing directly to txt doesn't buy us anything.